### PR TITLE
doesn't seem like EmptyBody is documented?

### PR DESCRIPTION
**Pull request by Enalmada**
# Pull Request Checklist

* [ ] Have you read [How to write the perfect pull request](https://github.com/blog/1943-how-to-write-the-perfect-pull-request)?
* [ ] Have you read through the [contributor guidelines](https://www.playframework.com/contributing)?
* [ ] Have you signed the [Lightbend CLA](https://www.lightbend.com/contribute/cla)?
* [ ] Have you [squashed your commits]? (Optional, but makes merge messages nicer.)
* [ ] Have you added copyright headers to new files?
* [ ] Have you checked that both Scala and Java APIs are updated?
* [ ] Have you updated the documentation for both Scala and Java sections?
* [ ] Have you added tests for any changed functionality?

# Helpful things

## Fixes

Fixes #xxxx

## Purpose

What does this PR do?

## Background Context

Why did you take this approach?

## References

Are there any relevant issues / PRs / mailing lists discussions?

### CODE REVIEW

File: documentation/manual/working/scalaGuide/main/ws/ScalaWS.md [hunk 1]
```diff
@@ -103,7 +103,7 @@ If you wish to specify a request timeout, you can use [`withRequestTimeout`](api
 
 ### Submitting form data
 
-To post url-form-encoded data a `Map[String, Seq[String]]` needs to be passed into `post`.
+To post url-form-encoded data a `Map[String, Seq[String]]` needs to be passed into `post`. (If the body is empty, you must pass play.api.libs.ws.EmptyBody into the post method.)
```
Comment (PromanSEW) [inline documentation/manual/working/scalaGuide/main/ws/ScalaWS.md]: I think this sentence must be on the new line and without parens.

**Comment by Enalmada** [comment]
Updated to add newline and remove paren.

File: documentation/manual/working/scalaGuide/main/ws/ScalaWS.md [hunk 2]
```diff
@@ -104,6 +104,7 @@ If you wish to specify a request timeout, you can use [`withRequestTimeout`](api
 ### Submitting form data
 
 To post url-form-encoded data a `Map[String, Seq[String]]` needs to be passed into `post`.
+If the body is empty, you must pass play.api.libs.ws.EmptyBody into the post method.
```
Comment (PromanSEW) [inline documentation/manual/working/scalaGuide/main/ws/ScalaWS.md]: What about adding link to `EmptyBody`?
```
[`EmptyBody`](api/scala/play/api/libs/ws/EmptyBody.html)
```

**Comment by Enalmada** [comment]
Changed EmptyBody to be a more concise link

**Comment by PromanSEW** [comment]
Hmm... Travis build is failed. I did not find `EmptyBody` in documentation
https://www.playframework.com/documentation/2.6.x/api/scala/index.html#play.api.libs.ws.package

**Comment by marcospereira** [comment]
Forwardported to master here: 80b714827ec759a9f16ba673bfae1ae87f0946db

**Pull request merged**